meilisearch-http/src/helpers/normalize_slashes.rs

```diff
@@ -0,0 +1,87 @@
+///
```

Comment:
Could you add the original PR address here please?

Reply:
I will not put all the links, so I found that on the issue https://github.com/actix/actix-web/issues/959, and I put the link of gist on the file.